Week1/homework/queries.js

```diff
@@ -0,0 +1,64 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'world',
+});
+
+connection.connect(err => {
+  if (err) throw err;
+  console.log('Connected...');
+});
+
+function sendQuery(query) {
+  connection.query(query, (err, results, fields) => {
+    if (err) throw err;
+    for (i in results) {
+      console.log(results[i]);
+    }
+  });
+}
+
+// Case4.a: What are the names of the countries with population greater than 8 million?
+const case4A = 'SELECT Name, Population FROM country WHERE Population > 8000000;';
```

Comment:
👍 but no need of the ";" at the end

Reply:
> Hello,
> 
> This is a really good homework. The code is neat and all the queries are good.
> Congrats!
> You don't need to add the ';' when requesting queries with nodejs.

Hi @mogaleaf , 
Thank you so much for your kind words and review.